File: remove/component-service/project-service/internal/model/project.go
package model

type Project struct {
	Id     string `json:"id"`
	Client string `json:"client"`
	Sector string `json:"sector"`
	Name   string `json:"name"`
	Model  string `json:"model"`
	Desc   string `json:"desc"`
	State  string `json:"state"`
}

type Projects []Project

File: BusinessLayer/Models/Project.cs
using BusinessLayer.Models.Common;
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace BusinessLayer.Models
{
    [Table("Project")]
    public class Project : BaseEntity
    {

        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        [JsonProperty(PropertyName = "projectId")]
        public int PId { get; set; }
        [JsonProperty(PropertyName = "orderId")]
        public int OId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string Manager { get; set; }
        public string Client { get; set; }
        public string Sector { get; set; }
        public virtual IEnumerable<ProjectHistory> ProjectHistory { get; set; }
        public virtual IEnumerable<Version> Versions { get; set; }
    }
}
